app/src/main/java/co/nimblehq/compose/crypto/ui/screens/home/TrendingItem.kt

```diff
@@ -33,14 +34,20 @@ import co.nimblehq.compose.crypto.ui.theme.Style.textColor
 import co.nimblehq.compose.crypto.ui.uimodel.CoinItemUiModel
 import coil.compose.rememberAsyncImagePainter
 
+const val TestTagTrendingItemSymbol = "TrendingItemSymbol"
+const val TestTagTrendingItemCoinName = "TrendingItemCoinName"
+const val TestTagTrendingItemPriceChange = "TrendingItemPriceChange"
+
+@Suppress("LongMethod")
 @Composable
 fun TrendingItem(
+    modifier: Modifier,
```

Comment:
Modifier could have a default value to get rid of definition in preview func

```suggestion
    modifier: Modifier = Modifier,
```